platform/src/main/kotlin/uk/gov/gdx/datashare/config/ApiExceptionHandler.kt

```diff
@@ -230,6 +244,8 @@ class SupplierSubscriptionNotFoundException(message: String) : Exception(message
 
 class NoDataFoundException(message: String) : Exception(message)
 
+class EnrichmentFieldsNotValidForEventType(message: String) : Exception(message)
```

Comment:
I think the rest of these exceptions all end in `..Exception`, could you change the name here too please :)